pymedphys/_experimental/cube.py

```diff
@@ -15,10 +15,7 @@
 
 from pymedphys._imports import mpl_toolkits
 from pymedphys._imports import numpy as np
-from pymedphys._imports import plt, pydicom
-
-from scipy.interpolate import splev, splprep
-from scipy.optimize import basinhopping
+from pymedphys._imports import plt, scipy
```

Comment:
is the removal of pydicom from the import deliberate?

Reply:
Yup, since its only use was to provide the typing in your other comment.

Reply:
that's what i thought.  OK.